starboard/android/apk/app/src/main/java/dev/cobalt/media/MediaCodecBridge.java

```diff
@@ -957,22 +957,22 @@ private int queueSecureInputBuffer(
       int[] numBytesOfEncryptedData,
       int numSubSamples,
       int cipherMode,
-      int blocksToEncrypt,
-      int blocksToSkip,
+      int patternEncrypt,
+      int patternSkip,
       long presentationTimeUs) {
     resetLastPresentationTimeIfNeeded(presentationTimeUs);
     try {
       CryptoInfo cryptoInfo = new CryptoInfo();
       cryptoInfo.set(
           numSubSamples, numBytesOfClearData, numBytesOfEncryptedData, keyId, iv, cipherMode);
-
-      if (cipherMode == MediaCodec.CRYPTO_MODE_AES_CBC) {
-        cryptoInfo.setPattern(new Pattern(blocksToEncrypt, blocksToSkip));
-      } else if (blocksToEncrypt != 0 || blocksToSkip != 0) {
-        Log.e(TAG, "Pattern encryption only supported for 'cbcs' scheme (CBC mode).");
-        return MediaCodecStatus.ERROR;
+      if (patternEncrypt != 0 || patternSkip != 0) {
+        if (cipherMode == MediaCodec.CRYPTO_MODE_AES_CBC) {
```

Comment:
As discussed offline, we are trying to make the implementation as close to the Chromium one as possible, but not apply any functional changes to the existing implementation.

In this particular case, we should add `boolean usesCbcs = cipherMode == MediaCodec.CRYPTO_MODE_AES_CBC;` in front of `CryptoInfo cryptoInfo = new CryptoInfo();` and use `usesCbcs` here.